src/marqo/s2_inference/model_registry.py

```diff
@@ -441,12 +441,14 @@ def _get_hf_properties() -> Dict:
     HF_MODEL_PROPERTIES = {
             "hf/all-MiniLM-L6-v1":
                 {"name": "sentence-transformers/all-MiniLM-L6-v1",
+                 # "url" : "https://marqo-cache-sentence-transformers.s3.us-west-2.amazonaws.com/all-MiniLM-L6-v1/all-MiniLM-L6-v1.zip",
```

Comment:
Are we adding these in the future?

Reply:
yes, as a back up url. let me remove these now. we can add later